bin/plugin/commands/packages.js

```diff
@@ -297,59 +336,64 @@ async function prepareForPackageRelease( minimumVersionBump, isPrerelease ) {
 		abortMessage
 	);
 
-	await updatePackages(
+	const commitHash = await updatePackageChangelogs(
 		gitWorkingDirectoryPath,
 		minimumVersionBump,
-		isPrerelease,
 		abortMessage
 	);
 
-	// Push the local changes
-	abortMessage = `Aborting! Make sure to push changes applied to WordPress release branch "${ releaseBranch }" manually.`;
 	await runPushGitChangesStep(
 		gitWorkingDirectoryPath,
 		releaseBranch,
 		abortMessage
 	);
 
-	abortMessage = 'Aborting! The release is finished though.';
-	await runCleanLocalFoldersStep( temporaryFolders, abortMessage );
-}
+	abortMessage = `Aborting! The release branch is synced. Make sure to publish the npm packages manually and backport the changelog commit (${ formats.title(
+		commitHash
+	) }) to master.`;
 
-async function prepareLatestDistTag() {
-	log(
-		formats.title(
-			'\n💃 Time to publish WordPress packages to npm 🕺\n\n'
-		),
-		'Welcome! This tool is going to help you with preparing everything for publishing a new stable version of WordPress packages.\n',
-		"To perform a release you'll have to be a member of the WordPress Team on npm.\n"
+	await publishPackages(
+		gitWorkingDirectoryPath,
+		minimumVersionBump,
+		abortMessage
 	);
 
-	await prepareForPackageRelease( 'patch' );
+	abortMessage = `Aborting! The packages have published though. Make sure to backport the changelog commit (${ formats.title(
+		commitHash
+	) }) to master.`;
 
-	log(
-		'\n>> 🎉 WordPress packages are ready to publish!\n',
-		'You need to run "npm run publish:prod" to release them to npm.\n',
-		'Let also people know on WordPress Slack when everything is finished.\n'
+	await runPushGitChangesStep(
+		gitWorkingDirectoryPath,
+		releaseBranch,
+		abortMessage
+	);
+
+	console.log(
+		`Packages published successfully. Make sure to backport the changelog commit (${ formats.title(
+			commitHash
+		) }) to master.`
 	);
+
+	abortMessage = 'Aborting! The release is finished though.';
+	await runCleanLocalFoldersStep( temporaryFolders, abortMessage );
 }
 
-async function prepareNextDistTag() {
+async function publishLatestPackages() {
 	log(
 		formats.title(
 			'\n💃 Time to publish WordPress packages to npm 🕺\n\n'
 		),
-		'Welcome! This tool is going to help you with preparing everything for publishing a new RC version of WordPress packages.\n',
+		'Welcome! This tool is going to help you release a new stable version of WordPress packages.\n',
 		"To perform a release you'll have to be a member of the WordPress Team on npm.\n"
 	);
 
-	await prepareForPackageRelease( 'minor', true );
+	await prepareForPackageRelease( 'patch' );
```

Comment:
We use `minor` way more often. `patch` is going to be useful during WordPress beta and RC. We should either offer two commands or ask the person doing release:

```sh
$ What do you want to do:
$ [ x ] Regular npm release ("minor" version)
$ [ ] Apply bug fixes only ("patch" version) 
$ [ ] Apply security patches to WordPress ("patch" version, "patch" dist-tag)

Reply:
The difference though is that we usually do regular npm release from the last Gutenberg plugin release using `minor` version bump.

When WordPress `trunk` is frozen, we don't sync from the last Gutenberg plugin release, but rather we cherry-pick changes to `wp/trunk` branch directly instead and do `patch` version bump.